doc/how_to_contribute/how_to_write_how_to_guides.rst

```diff
@@ -0,0 +1,47 @@
+How-To Write a How-To Guide
```

Comment:
Is there a specific format you want to enforce here? Such as "virtual sections" that each how-to needs and what should be in those? Such as a title block followed by a brief description of what the how-to covers. Then a section on requirements? And when to use subsections versus simply bolded headers?

Reply:
I added templates to these guides.  lmk what you think?